bob/src/lib.rs

```diff
@@ -6,6 +6,23 @@
 
 //! Library requires tokio runtime.
 
+#[cfg(all(
+    feature = "mimalloc",
+    target_arch = "x86_64",
+    target_env = "musl",
+    target_pointer_width = "64"
+))]
+use mimalloc::MiMalloc;
+
+#[cfg(all(
+    feature = "mimalloc",
+    target_arch = "x86_64",
+    target_env = "musl",
+    target_pointer_width = "64"
+))]
+#[global_allocator]
+static GLOBAL: MiMalloc = MiMalloc;
```

Comment:
`mimalloc` must be added at the executable level, i.e. in `bob-apps/src/bobd.rs`

Reply:
Done.